riklet/src/core.rs

```diff
@@ -112,8 +125,9 @@ impl Riklet {
         Ok(())
     }
 
+    #[tracing::instrument(skip(self), fields(instance_id = %instance_id, status = %status))]
     async fn send_status(&self, status: InstanceStatus, instance_id: &str) -> Result<()> {
-        event!(Level::DEBUG, "Sending status : {}", status);
+        debug!("Update instance status");
```

Comment:
I don't think this log is explicit enough. Maybe print the status ?

Reply:
Instrumentation provided by `tracing` makes it easier to handle logging. The method contains a derive `#[tracing::instrument(.., fields(instance_id, status))]`. For each log given in the current span of the method, it will print out fields provided by the prototype of the method. 

Here, you'll expect something similar to:

```
[TIMESTAMP] riklet::core: Update instance status instance_id=my-instance-123 status=ResourceStatus::Creating
```